packages/inputs-minimal/src/textfield/index.jsx

```diff
@@ -20,6 +20,7 @@ const TextField = ({
   disabled,
   onFocus,
   onBlur,
+  __type,
```

Comment:
```suggestion
  __type = 'text',
```

remove `defaultProps`